README.md

```diff
@@ -36,10 +36,10 @@ Options:
 Additional information:
   [path-to-fsh-defs]
     Default: "."
-    If fsh/ subdirectory present, it is included in [path-to-fsh-defs]
+    If input/fsh/ subdirectory present, it is included in [path-to-fsh-defs]
   -o, --out <out>
     Default: "build"
-    If fsh/ subdirectory present, default output is one directory above fsh/
+    If input/fsh/ subdirectory present, default output is one directory above fsh/
```

Comment:
Maybe this isn't a huge concern, because we are planning on adding a `generated` folder, and you are planning on adding that `generated` folder, but the way this currently works, `input` is treated as the output directory, which means the true output of SUSHI goes into `input/input/...`. Is it worth changing that now just so you can change it again when you add a `generated` folder? Maybe not, since we are merging into a feature branch.

Reply:
Ah yeah I was trying to keep out the changes needed for output change, but I guess I hadn't really thought about how weird the output would be in this case. I had output it to a different directory when testing so hadn't run into this issue, so now I guess I am a little more uncomfortable with how this works as an intermediate solution. But it is still intermediate I guess.

The other possible solution to this that I had considered was putting the output and input changes into one PR. Do you think that would make more sense?

(I haven't actually come to a conclusion here... shhh.)

Reply:
I don't think it is too weird to leave for now, since we are merging this into a branch that is not `master`, but if you were planning on doing the `input/generated` task next, then I'm also fine with wrapping that PR into this one. But really doesn't matter to me, I'll leave it to your preference.